Nodejs/Setup/NodejsTools/NodejsTools.wxs

```diff
@@ -15,6 +15,7 @@
             <Directory Id="Dir_$(var.key)TestFrameworks" Name="TestFrameworks">
                 <Directory Id="Dir_$(var.key)TestFrameworksExportRunner" Name="ExportRunner" />
                 <Directory Id="Dir_$(var.key)TestFrameworksMocha" Name="Mocha" />
+                <Directory Id="Dir_$(var.key)TestFrameworksTape" Name="Tape" />
```

Comment:
Also you mentioned you were unable to build the installer? What error were you running into?